tokio-trace-core/src/lib.rs

```diff
@@ -4,6 +4,45 @@
 #[macro_use]
 extern crate lazy_static;
 
+#[macro_export]
+macro_rules! metadata {
+    (
```

Comment:
What is the difference between these two macro branches?

Reply:
This is just so that the macro doesn't care if a trailing comma is present or not. I thought it was valuable for the macro to not break based on coding style, but I can remove this if you prefer.

Reply:
Oh! I completely missed that.

I'm whatever on it. If you want to leave it, ok. It doesn't seem necessary though as the only consumer of this will be a macro in `tokio-trace`, correct? I would probably remove one of the branches.

Reply:
I was operating under the assumption that anything publically exported to be consumed by `tokio-trace` ought to be written with the expectation that it was public API that others _might_ use, at least when it isn't `doc(hidden)`. I wanted to be open to an alternative implementation on top of `core` that also wanted to use this macro, so I figured it shouldn't be flaky if it's easily avoidable...